shared/schemas/custom-projects/create-custom-project.schema.ts

```diff
@@ -188,7 +188,7 @@ export const ValidateConservationSchema = (
     if (!params.projectSpecificLossRate) {
       ctx.addIssue({
         code: z.ZodIssueCode.custom,
-        message: "Project Specific Loss Rate is required",
+        message: "required",
```

Comment:
This seems to be potentially evil, why do we need to change the behavior? It seems to be to fit some test case